r2/r2/templates/prefoptions.html

```diff
@@ -128,11 +128,13 @@
           km of me</li>
       </ul>
 
-      <h6>${_("Personal Link")}</h6>
+      <h6><!-- doesn't look good: <a href="http://effectivealtruismhub.com/user/profiles">-->EA Profiles<!--</a>--> Link</h6>
```

Comment:
Yep, I deliberately left the comment to prevent someone else trying the same thing - it seemed worth it, notwithstanding its inelegance!